webapp/home/labs/genome/annotation.yml

```diff
@@ -135,6 +135,136 @@ tabs:
         view_link: ''
         view_tip: ''
         button_tip: Run in Galaxy AU
+        
+      subsections:
+       - id: TSI transcripts
+          title: Transcript alignment 
+          content:
+            - title_html: About these workflows
+              description_html: >
+                <p>
+                  This <a href="https://australianbiocommons.github.io/how-to-guides/genome_annotation/Transcriptome" target="_blank"> How-to-Guide </a> will describe the steps required to align transcript data to your genome on the Galaxy Australia platform, using multiple workflows. The outputs from these workflows can then be used as inputs into the next annotation workflow using FgenesH++.
+                </p>
+            - title_html: Repeat masking
+              description_html: >
+                <p>
+                  Mask repeats in the genome.
+                </p>
+              inputs:
+                - datatypes:
+                    - fasta
+                  label: Assembled genome <em>genome.fasta</em>
+              button_link: "{{ galaxy_base_url }}/workflows/trs_import?trs_server=workflowhub.eu&run_form=true&trs_id=875"
+              view_link: https://workflowhub.eu/workflows/875
+              view_tip: View in WorkflowHub
+              button_tip: Import to Galaxy Australia
+
+            - title_html: QC and trimming of RNAseq
+              description_html: >
+                <p>
+                  Trim and merge RNAseq reads.
+                </p>
+              inputs:
+                - datatypes:
+                    - fastqsanger.gz
+                  label: For each tissue: RNAseq R1 files in a collection <em>R1.fastqsanger.gz</em>; RNAseq R2 files in a collection <em>R2.fastqsanger.gz</em>
```

Comment:
This creates a syntax error - `For each tissue: `. Wrap the string in double quotes to get around this (two instances of this).